cmd/build-extensions-container.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"github.com/coreos/coreos-assembler/internal/pkg/cosash"
+	"path/filepath"
+	"io/ioutil"
+)
+
+func buildExtensionContainer()  error {
+	fmt.Println("Calling cmdlib")
+	sh, err := cosash.NewCosaSh()
+	if err != nil {
+		return err
+	}
+	if _, err := sh.PrepareBuild(); err != nil {
+		return err
+	}
+    sh.Process("runvm -- /usr/lib/coreos-assembler/build-extensions-oscontainer.sh $tmp_builddir/output.txt")
```

Comment:
Just going to call out I think this `CosaSh` thing is demonstrating its value here in allowing us to reuse all the shell script logic.